example/Testing.hs

```diff
@@ -10,7 +10,6 @@
 For testing the circuit notation.
 -}
 
-{-# LANGUAGE Arrows                    #-}
```

Comment:
I think this is the cause of the issue:
```
{-# LANGUAGE CPP            #-}
#if __GLASGOW_HASKELL__ <= 902
{-# LANGUAGE Arrows            #-}
#endif
```

I think fixes it for me. I'll try to test it on our codebase to see if there's any other issues.

Reply:
Any news on this?

Reply:
I set off a build and then forgot about it. That one failed because there were other changes since and we weren't up to date. I've set off another build with a fix, hopefully I won't forget about this one.